src/ExceptionType.cc

```diff
@@ -276,7 +276,7 @@ JSObject *ExceptionType::toJsError(JSContext *cx, PyObject *exceptionValue, PyOb
       JS::RootedString filename(cx, JS_NewStringCopyZ(cx, PyUnicode_AsUTF8(fileName)));
       JS::RootedString message(cx, JS_NewStringCopyZ(cx, msgStream.str().c_str()));
       // stack argument cannot be passed in as a string anymore (deprecated), and could not find a proper example using the new argument type
-      if (!JS::CreateError(cx, JSExnType::JSEXN_ERR, nullptr, filename, lineno, 0, nullptr, message, JS::NothingHandleValue, &rval)) {
+      if (!JS::CreateError(cx, JSExnType::JSEXN_ERR, nullptr, filename, lineno, JS::ColumnNumberOneOrigin(1), nullptr, message, JS::NothingHandleValue, &rval)) {
```

Comment:
Why no longer 0?

Reply:
`JS::ColumnNumberOneOrigin(0)`

**One Origin**

Reply:
`JS::CreateError()`'s function signature changed to `CreateError(..., JS::ColumnNumberOneOrigin column,...)`


See https://hg.mozilla.org/mozilla-central/file/d25c9bd/js/public/ErrorReport.h#l537

Reply:
`JS::ColumnNumberOneOrigin` is defined in https://hg.mozilla.org/mozilla-central/file/d25c9bd/js/public/ColumnNumber.h#l324